drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/panel/AnalysisReportScreenViewImpl.java

```diff
@@ -107,11 +107,27 @@ public void clearIssue() {
     }
 
     @Override
+    public void showStatusComplete() {
```

Comment:
It would probably be worth ensuring these colours are in keeping with [PatternFly](https://www.patternfly.org/pattern-library/widgets/#alerts).. and even use PatternFly/BS3 "alerts"?